test/commands.test.ts

```diff
@@ -295,7 +299,17 @@ suite('Command Handler', () => {
                 expect(err).equals(errorStatus.message);
             }
         });
-    });
+
+        test('wont remove if user does not confirm', async () => {
+          sandbox.stub(vscode.window, 'showWarningMessage').resolves();
+          try {
+            await handler.removeServer();
+            expect.fail();
+          } catch (err) {
+            expect(removeStub).not.calledOnce;
```

Comment:
This assertion will pass even if the method was called more than once.
If you only want to assert it was not called, use ```not.called```